src/components/AppNavigation/EditCalendarModal.vue

```diff
@@ -24,7 +24,11 @@
 					:placeholder="$t('calendar', 'Calendar name …')"
 					@input="calendarNameChanged = true">
 			</div>
-
+			<template v-if="canBeShared">
+				<NcCheckboxRadioSwitch :checked.sync="isTransparent">
+					{{ $t('calendar', 'All events in this calendar show you as busy by default') }}
```

Comment:
There is a logic error here: The state is inverted.

Checking means that we want an opaque calendar but will cause a transparent calendar and vice versa.

Either invert the wording or rename the variable to `isOpaque`.

Reply:
Is there? `this.isTransparent = calendar.transparency !== 'opaque'`